api/src/adapters/sam_gov/config.py

```diff
@@ -0,0 +1,26 @@
+"""Configuration for SAM.gov API client."""
+
+from pydantic import Field
+from pydantic_settings import SettingsConfigDict
+
+from src.util.env_config import PydanticBaseEnvConfig
+
+
+class SamGovConfig(PydanticBaseEnvConfig):
+    """Configuration for SAM.gov API client."""
+
+    model_config = SettingsConfigDict(env_prefix="SAM_GOV_", populate_by_name=True, extra="ignore")
```

Comment:
Huh, didn't know about populate_by_name, although it seems like Pydantic recommends not using it in favor of `validate_by_name` and `validate_by_alias` instead:
https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.populate_by_name